eng/pipelines/dotnet-buildtools-prereqs-eng.yml

```diff
@@ -4,9 +4,11 @@ pr:
     include:
     - main
   paths:
-    exclude:
-    - src/*
-    - README.md
+    include:
```

Comment:
exclude was originally used as it was viewed as the safer approach.  Meaning we would rather waste compute than skip running the validation on a new unexpected root file.  I don't feel strongly about this but felt I should point it out.

   exclude:
    - src/*
    - *.md
    - .git*
    - CODEOWNERS
    - LICENSE